src/apps/users/api/views.py

```diff
@@ -70,3 +75,20 @@ def unfollow(self, request: Request, *args, **kwargs):
             return Response("You are not following this user", status.HTTP_226_IM_USED)
         request.user.following.remove(user)
         return Response("Successfully")
+
+    @action(detail=False, methods=["GET"])
+    def verify(self, request: Request, *args, **kwargs):
+        token = request.GET.get("token")
```

Comment:
if u would use kwargs instead of params, this token would be in view params:

i mean if your endpoint would be `/api/v1/users/verify/<token>/`, then you will get token in verify method like this:
```python
def verify(self, request, token):
    ...
```